components/ReviewTransactionModal.tsx

```diff
@@ -0,0 +1,125 @@
+import useNotifications from "@/hooks/useNotifications";
```

Comment:
_(arbitrary comment location to start a thread)_

I'm seeing this error in the console 
<img width="1273" alt="image" src="https://user-images.githubusercontent.com/17163988/214171953-52a7e7f7-2145-4481-875b-02df7daa04b0.png">

Reply:
This happens as soon as I open the app. Tracking in https://github.com/ScopeLift/railgun-shielder/issues/63, we'll see if #62 fixes it and if not can revisit in a separate PR